internal/server/worker.go

```diff
@@ -115,8 +114,8 @@ type Worker struct {
 	*store.Worker
 
 	activeConnectionCount uint32 `gorm:"-"`
-	apiTags               []*Tag `gorm:"-"`
-	configTags            []*Tag `gorm:"-"`
+	apiTags               Tags
```

Comment:
I guess we currently never use the worker struct directly with `gorm` since its only being populated from the aggregate, but these will need to be exported when we do since you are dropping the gorm:"-" now we could do now or later just calling it out.

Reply:
Thanks! I've got a PR pending on top of this that takes care of that